src/routes/MetaDetails/MetaDetails.js

```diff
@@ -2,35 +2,33 @@
 
 const React = require('react');
 const PropTypes = require('prop-types');
-const { VerticalNavBar, HorizontalNavBar, MetaPreview, Image, useInLibrary } = require('stremio/common');
+const UrlUtils = require('url');
+const { VerticalNavBar, HorizontalNavBar, MetaPreview, ModalDialog, Image, useInLibrary } = require('stremio/common');
 const StreamsList = require('./StreamsList');
 const VideosList = require('./VideosList');
 const useMetaDetails = require('./useMetaDetails');
+const useMetaExtensions = require('./useMetaExtensions');
 const styles = require('./styles');
 
-const MetaDetails = ({ urlParams, queryParams }) => {
+const MetaDetails = ({ urlParams }) => {
     const metaDetails = useMetaDetails(urlParams);
+    const { tabs, selected, clearSelected } = useMetaExtensions(metaDetails.meta_resources);
```

Comment:
must be renamed to selectedMetaExtention.
just selected is not descriptive enough in this scope